src/nuget/Microsoft.DotNet.BuildTools.nuspec

```diff
@@ -20,6 +20,7 @@
       <dependency id="Microsoft.DotNet.TestHost" version="1.0.3-prerelease" />
       <dependency id="Microsoft.DotNet.CoreCLR" version="1.0.0-prerelease" />
       <dependency id="Microsoft.DotNet.BuildTools.ApiTools" version="1.0.0-prerelease" />
+      <dependency id="NuProj" version="0.10.4-beta-gf7fc34e7d8" />
```

Comment:
Hi Wes, do you mean the NuProj binary and targets should be embedded in this package, or the dependency should be implicit and the consumers responsible for adding it?  What's your thinking here?